package.json

```diff
@@ -126,7 +126,7 @@
     "redux-mock-store": "1.5.3",
     "snyk": "^1.161.1",
     "source-map-explorer": "1.8.0",
-    "typescript": "3.5.3"
+    "typescript": "3.7.5"
```

Comment:
Typescript is now at version: 3.8.3

Reply:
Ok, will do a proper testing again with the new version.